code/game/objects/items/cards_ids.dm

```diff
@@ -305,6 +300,9 @@
 /obj/item/card/id/GetID()
 	return src
 
+/obj/item/card/id/RemoveID()
+	return src
```

Comment:
Wait, shouldn't this remove the ID?

Reply:
it's what PDA does, not a card does.
```
/obj/item/pda/RemoveID()
	return do_remove_id()
```
I think it's trying a parent trick, so that you remove id from PDA properly, but not remove id from itself.
I know it looks weird, but it's how ID card should work.

Reply:
yeah this is for PDAs, tablets, and wallets

Reply:
Oh I thought this was the Id computer